### DISCUSSION TITLE
Setup instructions are misleading?

### DISCUSSION THREAD

**Discussion by chris-hatton** [grpc/grpc-kotlin#308] [Q&A]
As a relative newcomer to gRPC I found the [setup instructions here](https://github.com/grpc/grpc-kotlin/blob/master/compiler/README.md) are potentially misleading?
Consider the advice to set-up the Gradle plugin:
```
...
generateProtoTasks {
    all().forEach {
        it.plugins {
            id("grpc")
            id("grpckt")
        }
        it.builtins {
            id("kotlin") // <-- Note single `kotlin` built-in here
        }
    }
}
...    
```
I found that while, yes, this setup causes the **gRPC-Kotlin** plugin to do it's own job of creating **Kotlin Wrappers** for Java classes, **the depended-on Java classes are not generated** until one also includes the 'original' built-in `id("java")` - so that for a complete working gRPC code generation, it's necessary to see `builtins` configured as:
```
...
it.builtins {
    id("java")
    id("kotlin")
}
...
```
Is this expected? If so, it should really be called out in the documentation that the Java built-in is still needed for a working setup even if that configuring that part itself it outside the scope of `grpc-kotlin`'s documentation.
If this isn't expected and the `kotlin` plugin should be calling through to **Java** generation implicitly; why might it fail to?

**Comment by jamesward** [comment]
Hmmm... I don't think I had to add the `id("java")` one as it is there by default.  See for example:
https://github.com/grpc/grpc-kotlin/blob/master/examples/stub/build.gradle.kts

I'm not sure why we are experiencing a different behavior.  What else might be different between our uses?
